ui/components/component-library/button-base/button-base.scss

```
@@ -0,0 +1,80 @@
+.mm-button {
+  position: relative;
+  height: 40px;
+  padding: 0;
+  border-radius: 999px;
+  cursor: pointer;
+  color: var(--color-text-default);
+  background-color: #dcdcdc;
+  vertical-align: middle;
+  user-select: none;
+
+  &:active,
+  &:hover {
+    color: var(--color-text-default);
+  }
+
+  &--block {
+    display: block;
+    width: 100%;
+  }
+
+  &--content {
+    height: 100%;
+  }
+
+
+  &--size-sm {
+    height: 32px;
+  }
+
+  &--size-md {
+    height: 40px;
+  }
+
+  &--size-lg {
+    height: 48px;
+  }
+
+  &--size-auto {
+    height: auto;
+    background-color: transparent;
+    border-radius: 0;
+    vertical-align: top;
+    font-family: inherit;
+    font-weight: inherit;
+    font-size: inherit;
+    line-height: inherit;
+    letter-spacing: inherit;
+  }
+
+  &--loading {
+    cursor: not-allowed;
+  }
+
+  &--loading .mm-button--content {
+    color: transparent;
+  }
+
+  &--disabled,
+  &:disabled {
+    opacity: 0.3;
+    cursor: not-allowed;
+  }
+}
+
+
+
+.mm-icon--loading {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  animation: spinner 1.2s linear infinite;
+}
```

Could this but changed to 

```suggestion
  }
  
  &__icon--loading {
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    animation: spinner 1.2s linear infinite;
  }
}
```